packages/eslint-plugin-bbva/lib/rules/no-throw-in-unchained-promise.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+module.exports = {
+	meta: {
+		type: 'suggestion',
+		docs: {
+			description: 'Avoid throwing error in unchained promise handlers',
+			recommended: false
+		},
+		fixable: null,
+		schema: []
+	},
+
+	create(context) {
+		return {
+			// Matches any `throw` inside a `then` / `catch` that is not assigned to a variable or returned
+			[':not(:matches(ReturnStatement, VariableDeclarator, AssignmentExpression)) >' +
+			'CallExpression:matches([callee.property.name=then], [callee.property.name=catch]) ThrowStatement'](node) {
```

Comment:
Brackets are not needed here.

Reply:
Yep, it is, notice there is a `+` operator joining the two strings. I didn't like the multiline approach using template literals, as the final string would contain the tab characters. Any other formatting suggestions are very welcome here :)